packages/docs/getting-started/status.md

```diff
@@ -1,129 +0,0 @@
-# UI Status
```

Comment:
From what we've been hearing, this page is useful for our developers to know what state these components are in. Why does this team feel like it should be removed?

Reply:
All of our components are in beta, and all new and existing components will continue to be in beta through the FY21 roadmap (@edburyenegren-okta keep me honest here). 

Do you know what they find useful about it? If we had different platforms/libraries I could see this being useful right now. Similar to the changelog, we are going to re-evaluate this and likely bring it back as part of the next major iteration of the docs site.

Reply:
This warrants a larger discussion - am fine with removing this now as we work some things out.